scripts/azure-pipelines/install-environment.ps1

```diff
@@ -0,0 +1,39 @@
+# Ensures we have downloaded and extracted a build environment
+# into our Azure Pipelines VM before we attempt to build. If the
+# environment was already cached and hence already present
+# this script will detect that.
+#
+# Configuration variables used from environment:
+#
+#  MUMBLE_ENVIRONMENT_VERSION - Full build environment version
+#                               (e.g. win64-static-no-ltcg-1.3.x-2017-02-02-ec94ddb-790).
+#                               Must match .7z and extracted folder name.
```

Comment:
From where? Where are those defined?
So, should mention the one below. Maybe swap the order around.

Reply:
Done.